src/debug/HyprCtl.cpp

```diff
@@ -789,9 +791,12 @@ std::string bindsRequest(eHyprCtlOutputFormat format, std::string request) {
                 ret += "e";
             if (kb.nonConsuming)
                 ret += "n";
+            if (kb.hasDescriptions) {
```

Comment:
no {}

Reply:
fixed in commit 05a1e22ed93afefb84b19bec05ed66d3de2e568b